### PR TITLE
Upgrading to 2018.3.0b7

### PR DESCRIPTION
**Pull request by jessicafalk**
**Contributions**: We are not currently taking public contributions - see our [contributions](CONTRIBUTING.md) policy. However, we are accepting issues and we do want your [feedback](../README.md#give-us-feedback).

-------

#### Description
Tried upgrading to latest beta. They did some changes to the prefab so had to re-save all of them.
To check it out: https://console.improbable.io/projects/unity_gdk/deployments/test20183beta/overview

#### Tests
ran local and cloud deployments on both mac and windows.
linux builds work when started from command line 
#### Documentation
wip - will add, if people are ok with upgrading

#### Primary reviewers
If your change will take a long time to review, you can name at most two primary reviewers who are ultimately responsible for reviewing this request. @ mention them.

### CODE REVIEW

**Comment by petemounce** [comment]
Suggestion: update the version inside the powershell setup script as well?

**Comment by jessicafalk** [comment]
will do, if we decide to ugprade

**Comment by oblm** [comment]
**Testing:** I'm checking out this branch now in order to check if this changes is backwards compatible with 2018.2.0f2, our current recommended version.

**Docs:** Goes without saying we should document this change.

File: workers/unity/ProjectSettings/ProjectSettings.asset [hunk 1]
```diff
@@ -631,17 +576,37 @@ PlayerSettings:
...
+    m_VersionName: 
   facebookSdkVersion: 7.9.4
-  apiCompatibilityLevel: 3
+  facebookAppId: 
+  facebookCookies: 1
+  facebookLogging: 1
+  facebookStatus: 1
+  facebookXfbml: 0
+  facebookFrictionlessRequests: 1
+  apiCompatibilityLevel: 6
```
Comment (zeroZshadow) [inline workers/unity/ProjectSettings/ProjectSettings.asset]: Why was the api compatibility level changed?

File: workers/unity/Packages/manifest.json [hunk 2]
```diff
@@ -7,7 +7,8 @@
     "com.improbable.gdk.testutils": "file:com.improbable.gdk.testutils",
     "com.improbable.gdk.tools": "file:com.improbable.gdk.tools",
     "com.improbable.gdk.transformsynchronization": "file:com.improbable.gdk.transformsynchronization",
-    "com.unity.package-manager-ui": "1.9.11",
+    "com.unity.collab-proxy": "1.2.9",
```
Comment (zeroZshadow) [inline workers/unity/Packages/manifest.json]: Why is collab proxy added?

File: workers/unity/ProjectSettings/ProjectSettings.asset (hunk 1 above)
Comment (jessicafalk) [reply workers/unity/ProjectSettings/ProjectSettings.asset]: 🤷‍♀️  can try to revert it

File: workers/unity/Packages/manifest.json (hunk 2 above)
Comment (jessicafalk) [reply workers/unity/Packages/manifest.json]: unity

File: workers/unity/ProjectSettings/ProjectSettings.asset (hunk 1 above)
Comment (jessicafalk) [reply workers/unity/ProjectSettings/ProjectSettings.asset]: tried to revert it. didn't let me. manually set the api compatibility level for the platforms

File: workers/unity/Packages/manifest.json [hunk 3]
```diff
@@ -7,7 +7,7 @@
     "com.improbable.gdk.testutils": "file:com.improbable.gdk.testutils",
     "com.improbable.gdk.tools": "file:com.improbable.gdk.tools",
     "com.improbable.gdk.transformsynchronization": "file:com.improbable.gdk.transformsynchronization",
-    "com.unity.package-manager-ui": "1.9.11",
+    "com.unity.package-manager-ui": "2.0.2",
```
Comment (jamiebrynes7) [inline workers/unity/Packages/manifest.json]: Shiny

**Comment by jamiebrynes7** [comment]
Closing this as we aren't going to upgrade until stable!

**Pull request closed without merging**